components/tools/OmeroPy/src/omero/plugins/metadata.py

```diff
@@ -604,20 +604,22 @@ def pixelsize(self, args):
         else:
             raise Exception("Not implemented for type %s" % md.get_type())
 
+        ctx = {'omero.group': '-1'}
+
         params = omero.sys.ParametersI()
         params.addId(md.get_id())
-        pixels = client.getSession().getQueryService()\
+        pixels = client.getSession().getQueryService(ctx)\
                                     .findAllByQuery(q, params)
```

Comment:
I think the `ctx` needs to be passed to the `findAllByQuery`. On eel I'm getting the same behavior of no change. Likely `if not pixels:` should also call `self.ctx.die` or at least `self.ctx.err`.

Reply:
Argh, of course, thanks @joshmoore .